src/stryker/plugins.pug

```diff
@@ -30,17 +32,17 @@ block content
         * `clear-text`: A clear text report at the end of the mutation testing phase.
         * `event-recorder`: Write all events to disk. Useful for debug purposes (default location: 'reports/mutation/events')
 
-        The html reporter can be installed a separate dependency. It can generate [an html report](https://stryker-mutator.github.io/stryker-html-reporter).
-        For more info, see the [stryker-html-reporter readme](https://github.com/stryker-mutator/stryker/tree/master/packages/stryker-html-reporter#readme)
+        The html reporter can be installed a separate dependency. It can generate [an html report](https://stryker-mutator.io/stryker-html-reporter).
```

Comment:
```suggestion
        The html reporter can be installed a separate dependency. It can generate [a html report](https://stryker-mutator.io/stryker-html-reporter).
```